pytential/linalg/skeletonization.py

```diff
@@ -400,44 +480,39 @@ def __getitem__(self, i: int) -> Tuple[np.ndarray, np.ndarray]:
         return [pxymat_i, nbrmat_i]
```

Comment:
I thought I was promised a tuple! :)

Reply:
Haha, indeed! Fixed in https://github.com/inducer/pytential/compare/440161e74f559e51120245ce59f8c1a6c4f15d77..0617bdafd0a7a37c2e162b752371ab908e456ad5.